src/app.scss

```diff
@@ -0,0 +1,219 @@
+body {
+  overflow-x: hidden;
+}
+
+h2,
+h3,
+h4,
+p {
+  margin: 0;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.w-100 {
+  width: 100%;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-row {
+  flex-direction: row;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.container {
+  width: 100vw;
+  height: 100vh;
+  background-color: #f5f6fa;
+}
```

Comment:
- [ ] Good job with the design so far. However, I noticed that the heading of the app disappears when I added up to two books. After checking your CSS styling, I found out that the issue is caused by `height: 100vh;` given to the `.container`. Let's fix this issue by removing the `height: 100vh;` or making the `height: 100%;` so as to improve the UI of your app. 

When there is one book:
<img width="669" alt="Loom_qdr3K3stiI" src="https://user-images.githubusercontent.com/81468675/150614129-7e10b555-67bd-457b-94b4-195c942dd5b7.png">

When there are more than one book:
<img width="851" alt="Loom_u29Zo4fCVN" src="https://user-images.githubusercontent.com/81468675/150614650-47db987a-924d-46db-b83e-62e5aaec029a.png">